recipes/libuldaq/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+libtoolize
+autoreconf -i
+./configure --disable-examples --prefix=$PREFIX
+make -j"${CPU_COUNT}"
+make install
```

Comment:
Is there a `make check` that can be run?

Reply:
Sure! Fixed.